src/Authentication.Abstractions/AzureEnvironment.cs

```diff
@@ -231,6 +231,17 @@ private static AzureEnvironment MapArmToAzureEnvironment(ArmMetadata armMetadata
                 }
             }
 
+            if (!string.IsNullOrEmpty(armMetadata.LogAnalyticsResourceId))
+            {
+                string api_version = "v1";
+                var logAnalyticsEndpoint = armMetadata.LogAnalyticsResourceId + '/' + api_version;
+                azureEnvironment.SetProperty(ExtendedEndpoint.OperationalInsightsEndpointResourceId, armMetadata.LogAnalyticsResourceId);
+                if (!string.IsNullOrEmpty(logAnalyticsEndpoint))
```

Comment:
In this case, logAnalyticsEndpoint would not be null or empty. This conditional statement could be removed.

Reply:
Removing unnecessary conditional.